doc/how-to/manage-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+# Manage Infrastructure
```

Comment:
This is title case whereas the other file's `h1` and all other headings are sentence case